### DISCUSSION TITLE
Can this be configured to use an Asus Router? Can URLs be blocked?

### DISCUSSION THREAD

**Discussion by Mr-McMuffin** [serverless-dns/serverless-dns#193] [Q&A] [answered]
1) Can this be used on an asus router, i would like to test it but have no idea what to put in the IP field for rethinkdns.

2) Can URLs be manually blocked in rethinkdns, or will it be added in the future? 

Thanks.

**Comment by Mr-McMuffin** [comment]
anyone?

**Comment by ignoramous** [comment, marked as answer]
Hi,

re: 1

Rethink only supports *DNS over TLS* (DoT) and *DNS over HTTPS* (DoH). Find out if your router supports either of these (unlikely, unless you've flashed openWRT or other such FOSS OS for your router).

Most operating systems, like Windows, macOS, Android, iOS now support either DoT or DoH out-of-the-box.

---

re: 2

We intend to support "URL" blocking like uBlockOrigin and AdGuard apps / plugins can with [Rethink DNS + Firewall](https://github.com/celzero/rethink-app) (on Android), but it isn't something we've begun implementing, yet.

The server (DNS resolver) can't really block URLs, so that's a non-starter.
